app/assets/javascripts/views/vacation_request.js

```diff
@@ -0,0 +1,123 @@
+App.Views.VacationRequest = Backbone.View.extend({
+  el: 'section',
+  template: JST['templates/vacation_request'],
+
+  initialize: function() {
+    this.$el.html( this.template() );
```

Comment:
In Javascript extra space after '(' and before ')' not required, it's Ruby style. Please remove.